pcsx2/Hw.cpp

```diff
@@ -65,12 +65,7 @@ void hwReset()
 	psHu32(DMAC_ENABLEW) = 0x1201;
 	psHu32(DMAC_ENABLER) = 0x1201;
 
-	if ((psxHu32(HW_ICFG) & (1 << 3)))
-	{
-		SPU2ps1reset();
-	}
-
-	SPU2reset();
+	SPU2reset(false);
```

Comment:
Looks like this one is still using the boolean and somehow gets away with it?

Reply:
you're right, i forgot my git add, was about to do a second pass but as i'm multitasking, i was waiting for builds to run first; this is because the enum in this case is casted as a bool